back/app/services/xlsx_service.rb

```diff
@@ -127,6 +127,8 @@ def generate_users_xlsx(users, view_private_attributes: false)
       { header: 'last_name', f: ->(u) { u.last_name } },
       { header: 'profile_page', f: ->(u) { url_service.model_to_url(u) }, skip_sanitization: true },
       { header: 'created_at', f: ->(u) { u.created_at }, skip_sanitization: true },
+      { header: 'registration_completed_at', f: ->(u) { u.registration_completed_at }, skip_sanitization: true },
```

Comment:
Why do you need `skip_sanitization`? I wouldn't use it by default.

Reply:
Because it seems to be skipped on every other similar field in this file (i.e. Date fields that are generated programatically - not from user input).

Reply:
Makes sense :+1: